### DISCUSSION TITLE
Generate Traces - function or class?

### DISCUSSION THREAD

**Discussion by beckydvn** [AI-Planning/macq#38] [General]
In the original Usage, the generated traces were handled in a `Generate` class. However, considering the idea that random trace generation is just one of many trace generation techniques, it probably makes more sense to use a series of functions instead. (Especially since Python doesn't support multiple constructors...) Thoughts?

**Comment by haz** [comment]
Sounds good to me.

@alisonparedes : Can you think of any reason we'd like to keep things Class/object-centric for the generation?
